apps/zbugs/schema.ts

```diff
@@ -189,26 +189,16 @@ type AuthData = {
   role: 'crew' | 'user';
 };
 
-export const schema = createSchema(
-  5,
-  {
-    user,
-    issue,
-    comment,
-    label,
-    issueLabel,
-    viewState,
-    emoji,
-    userPref,
-  },
-  {
+export const schema = createSchema(5, {
+  tables: [user, issue, comment, label, issueLabel, viewState, emoji, userPref],
```

Comment:
What happens if there are duplicate table names in here? Is there a runtime error?